### ISSUE TITLE
"import non blocking" message?

### ISSUE THREAD

**Issue by aroberge** [brython-dev/brython#664]
I was testing a program with something like

    from my_module import (function_1, function_2
                         function_3, function_4)
    # more code follows
and was puzzled to see that the program ended immediately after this import statement. Looking at the console, I saw the following message

    import non blocking my_module

If I remove the parentheses to write everything on the same line

    from my_module import function_1, function_2, function_3, function4
    # more code follows

my program executes correctly (and there is not message in the console about non blocking import.
This is using

    __BRYTHON__.__MAGIC__ == "3.2.7"

It this so-called non-blocking import an expected behaviour or a bug?

**Comment by PierreQuentel** [comment]
I installed version 3.2.7 and tried to reproduce the bug but I don't have this error ; if you have it it's a bug, not a feature.
It comes from a time when I tried to implement a non-blocking version of `import`. It requires wrapping all the code after the import in a function that is executed when the import is done. Roughly speaking,

```python
import some_module
print('ok')
```
is translated to something like
```
$import("some_module",
    function(){
        print('ok')
    }
)
```
Unfortunately, this approach doesn't work with the built-in `__import__` function, which can be aliased with any random name, and it also doesn't work for imports inside a function :

```python
def f():
    import some_module
    print('message 1')

f()
print('message 2')
```
In standard Python you expect to get "message 1" before "message 2" ; to achieve this with non-blocking calls, the code after the call `f()` should also be wrapped in a function...

So I have given up this idea, but probably forgot to remove some pieces of code in __py_import.js__, hence the strange error message you got.

I just hope that synchronous Ajax calls remain deprecated but not withdrawn, otherwise it's more or less the end of Brython :-(
